eng/Versions.props

```diff
@@ -68,7 +68,7 @@
       Source-build builds the product with the most recent previously source-built release. Thankfully, these two requirements line up nicely
       such that any version that satisfies the VS version requirement will also satisfy the .NET SDK version requirement because of how we ship.
     -->
-    <MicrosoftCodeAnalysisVersion_LatestVS>4.7.0</MicrosoftCodeAnalysisVersion_LatestVS>
+    <MicrosoftCodeAnalysisVersion_LatestVS>4.8.0</MicrosoftCodeAnalysisVersion_LatestVS>
```

Comment:
Can you also introduce a property for 4.7 (which shipped with .NET 8.0.100)

Reply:
Correct me if I'm wrong but roslyn4.8 shipped with VS 17.8 and 8.0.100.

Reply:
> roslyn4.8 shipped with VS 17.8 and 8.0.100

Yes, roslyn 4.8 corresponds to VS 17.8 ([documentation](https://learn.microsoft.com/en-us/visualstudio/extensibility/roslyn-version-support?view=vs-2022)). That's the reason we are upgrading it in the first place - since VS 17.8 (roslyn 4.8) is the first version, which fully supports C# 12.

Do I still need to introduce property for 4.7 then? If so, where it should be used?

Reply:
Yep I was thinking of the release that first supported .NET 8.0.

We still need the separate property that's locked to 4.8 then (instead of 4.7) to support the netstandard2.0 packages that first shipped generators in 8.0 (in particular the options generator comes to mind).